Gemfile

```diff
@@ -8,6 +8,10 @@ gem "jbuilder", "~> 2.5"
 gem "jquery-rails"
 gem "slim"
 gem "font-awesome-rails"
+
+# NOTE: this is no longer needed in Rails 5. See
+# https://github.com/heroku/rails_stdout_logging#rails-5 for instructions on how
+# to transition.
 gem "rails_stdout_logging", group: [:development, :staging, :production]
```

Comment:
It shouldn't be a problem for now. But we should have a staging environment indeed.